src/components/CmsZone.vue

```diff
@@ -36,7 +36,7 @@
   </div>
 </template>
 
-<style>
+<style scoped>
```

Comment:
I don't think this is what we want. The `.scrollable` class here is a real abstraction piercing nightmare. Basically, the CMS zone provides a feature I call viewable impression tracking. It's a feature that only tracks an impression when the of the content is actually visible for some time (sweet implementation by Ilya). The implementation assumes that the scrolling parent of the zone has the class `scrollable`. I just put these styles here since the class was explicitly mentioned. Here's what I think we should do:

1. Move these styles into the FreshEBT app.
2. Add a `pluginOption` called `scrollableParent` with a default value of `body`.
3. Modify line 177 below to use `pluginOptions.scrollableParent` instead of '.scrollable-content`.
4. Set scrollableParent in FreshEBT to `.scrollable-content`.